backend/src/main/java/com/bakdata/conquery/resources/admin/rest/AdminDatasetProcessor.java

```diff
@@ -215,11 +233,12 @@ public synchronized void addConcept(@NonNull Dataset dataset, @NonNull Concept<?
 		if (datasetRegistry.get(dataset.getId()).getStorage().hasConcept(concept.getId())) {
 			throw new WebApplicationException("Can't replace already existing concept " + concept.getId(), Response.Status.CONFLICT);
 		}
+		final Namespace namespace = datasetRegistry.get(concept.getDataset().getId());
 
 
 		// Register the Concept in the ManagerNode and Workers
 		datasetRegistry.get(dataset.getId()).getStorage().updateConcept(concept);
```

Comment:
Du könntest das hinzufügen in den eigenen Store auch als Slowjob machen. Wir haben da leider keine semantik oder echte Regeln zu. Eines Tages ... :D

Reply:
bzw ich denke sogar, dass du das musst um den Bug zu verhindern, den du vorher beschrieben hast?

Reply:
Der Bug, den ich gemeint habe, trat nicht zum Glück nicht auf